sa/sa.go

```diff
@@ -150,8 +148,7 @@ func (ssa *SQLStorageAuthority) GetRegistrationByKey(ctx context.Context, key *j
 		model, err = selectRegistration(ssa.dbMap, query, sha)
 	}
 	if err == sql.ErrNoRows {
-		msg := fmt.Sprintf("No registrations with public key sha256 %s", sha)
-		return core.Registration{}, core.NoSuchRegistrationError(msg)
+		return core.Registration{}, berrors.NotFoundError("registration with key hash %q not found", sha)
```

Comment:
I think this message is less informative than the old one, please revert it. Also: please avoid making unrelated changes to error messages in a big change like this one. :-)